dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/data/handler/DataHandler.java

```diff
@@ -370,7 +376,7 @@ void addProgramDataElementAttributeIndicatorValues( DataQueryParams params, Grid
             EventQueryParams eventQueryParams = new EventQueryParams.Builder( fromDataQueryParams( dataSourceParams ) )
                 .withSkipMeta( true ).build();
 
-            Grid eventGrid = eventAnalyticsService.getAggregatedEventData( eventQueryParams );
+            Grid eventGrid = eventAnalyticsService.getAggregatedEventData( eventQueryParams, false );
```

Comment:
Could you instead embed the `tryCache` param part of the `EventQueryParams` object? That is meant to encapsulate parameters. Can be added below the `withSkipMetadata` line.

We could also name the field `skipCacheLookup` with default value of false. I think false is the most sensible value and avoids a breaking change.